test/test_utils.jl

```diff
@@ -1,3 +1,5 @@
+include("dependencies_for_runtests.jl")
+
```

Comment:
`runtests.jl` includes that and then `test_utils.jl` re-includes it?
what about function re-definitions? I think in Julia 1.7 we may be getting warnings for that.

Reply:
Dang

Reply:
The purpose here is to include the test file at the REPL so we can debug a test with `Revise`. Maybe there's a better way. Or, we can remove all of these, and put them in just when we need to do that debugging.